components/Timeline.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <section class="container">
+        <article class="section is-medium">
+            <div class="timeline">
+                <TimelineHeader
+                title="Début"
+                date="0000-00-00T00:00:00.000Z"
+                type="primary"
+                size="medium"
+                />
+
+                <div v-for="(item, index) in orderedItems" v-bind:key="index">
+                    <TimelineHeader
+                    v-if="typeOf(item) == 'timelineheader'"
+                    :title="item.timelineheader.title"
+                    :date="item.timelineheader.date"
+                    :dateFormat="item.timelineheader.dateFormat"
+                    :type="item.timelineheader.type"
+                    :size="'small'"
+                    />
+
+                    <TimelineItem
+                    v-else
+                    :title="item.timelineitem.title"
+                    :date="item.timelineitem.date"
+                    :dateFormat="item.timelineitem.dateFormat"
+                    :image="item.timelineitem.image"
+                    :icon="item.timelineitem.icon"
+                    :marker_type="item.timelineitem.marker_type"
+                    :item_type="item.timelineitem.item_type"
+                    />
+                </div>
+                
+                <TimelineHeader
+                title="Fin"
+                date="0000-00-00T00:00:00.000Z"
+                type="primary"
+                size="medium"
+                />
+            </div>
+        </article>
+    </section>
+</template>
+
+
+<script>
+export default {
+  props: {
+    items: { type: Array, required: true, default: () => new Array() },
+  },
+  methods: {
+    typeOf(item) {
+      return Object.keys(item)[0].toLowerCase();
+    },
+  },
+  computed: {
+    orderedItems: function(){
+      return this.items.sort((a, b) => {
+        if(a.timelineitem){a = a.timelineitem}else {a = a.timelineheader}
```

Comment:
Format correctly

Reply:
If you meant to format the IF block :
- https://github.com/BDE-Polytech-MTP/website-front/pull/56/commits/328f67a7e248295641d52c863d16362a38b8b9c4